src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -1300,6 +1300,20 @@ var ReactCompositeComponent = {
     return inst;
   },
 
+  toTree: function() {
+    const element = this._currentElement;
+    // not using `children`, but I don't want to rewrite without destructuring
+    // eslint-disable-next-line no-unused-vars
+    const { children, ...propsWithoutChildren } = element.props;
+    return {
+      nodeType: ReactNodeTypes.getType(element),
+      type: element.type,
+      props: propsWithoutChildren,
+      instance: this._instance,
+      rendered: this._renderedComponent.toTree(),
+    };
+  },
```

Comment:
hmm. This makes sense as a rule. I'm not sure what the cleanest way to do this is but I'll give it a shot